steps/firebase-change-file-by-env/0.0.1/step.yml

```diff
@@ -0,0 +1,83 @@
+title: Firebase Change File by environment
+summary: |
+  Change file change firebase file according to branch
+description: |
+  Change file between firebase project according to branch in your repository
+  * Important Suggestion *: You must create a folder in the root of your project called 'firebase_env'
+   and inside have two folders for production and development, the step suggests in its variables
+   and the corresponding folders.
+website: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env
+source_code_url: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env
+support_url: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env/issues
+published_at: 2021-03-31T01:05:02.040459-03:00
+source:
+  git: https://github.com/manuelabarca/bitrise-step-firebase-change-file-by-env.git
+  commit: 753198d247c8969aaf255ea08d62ac6d2a8b4fcd
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+type_tags:
+- utility
+inputs:
+- opts:
+    description: |
+      Branch usually used for the production environment like 'master' or 'main'
+    is_required: true
+    title: Principal branch for production
+  principal_branch: master
+- opts:
+    description: |
+      Choose the platform where you want to use: ios, android or both
+    is_expand: true
+    is_required: true
+    title: Platform to use
+    value_options:
+    - both
+    - ios
+    - android
+  platform: both
+- opts:
+    description: |
+      Working Directory of the step, default is for react-native project
+    title: Working directory for android platform
+  workdir_android: $BITRISE_SOURCE_DIR/android/
+- opts:
+    description: "Working Directory of the step, default is for react-native project
+      \n"
+    title: Working directory for ios platform
+  workdir_ios: $BITRISE_SOURCE_DIR/ios/
+- ios_scheme_name: $BITRISE_SCHEME
+  opts:
+    description: |
+      Scheme used, usually the name of your app on iOS like 'Example or MyApp'
+    is_required: true
+    title: Scheme name
+- android_module: app
+  opts:
+    description: Module name, It is usually the app folder inside the android folder
```

Comment:
is it a module or folder name?

Reply:
This is a folder name but usually called module in android app